djangobaseview/static/js/guestbook/GuestbookStore.js

```diff
@@ -0,0 +1,61 @@
+define([
+	"dojo/_base/declare",
+	"dojo/store/JsonRest",
+	"dojo/cookie",
+	"dojo/Stateful"
+], function(declare, JsonRest, cookie, Stateful){
+	return declare([Stateful],{
+		jsonRest: null,
+		guestbookName: null,
+
+		_guestbookNameGetter: function(){
+			return this.guestbookName;
+		},
+
+		_guestbookNameSetter: function(guestbook_name){
+			this.guestbookName = guestbook_name;
+		},
+
+		constructor: function(){
+			this.watch("guestbookName", function(name, oldValue, value){
+				if (oldValue !== value){
+					this.guestbookName = value;
+					this.jsonRest = new JsonRest({
+						target: "/api/guestbook/" + this.guestbookName + "/greeting/",
+						headers: {"X-CSRFToken": cookie("csrftoken")}
+					});
+				}
+			});
+		},
+
+		getGreetings: function(guestbook_name, cursor){
+			return this.jsonRest.query({
```

Comment:
với mỗi `guestbook_name` cần thay đổi  `this.jsonRest` cho phù hợp, hãy set lại giá trị this.guestbookName trước khi query